<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Rebot 6.0.1 (Python 3.10.4 on win32)" generated="20230523 14:56:48.293" rpa="false" schemaversion="3">
<suite id="s1" name="Main" source="G:\My Drive\Python\Scraper\Lelongtips\sherlock-dms\tests\restAPI\Main\Main.robot">
<test id="s1-t1" name="1 - Executes Main flow" line="6">
<kw name="When user runs main flow" library="Main">
<msg timestamp="20230523 14:56:45.971" level="TRACE">Arguments: [  ]</msg>
<msg timestamp="20230523 14:56:45.975" level="DEBUG">Starting new HTTPS connection (1): www.lelongtips.com.my:443</msg>
<msg timestamp="20230523 14:56:46.917" level="DEBUG">https://www.lelongtips.com.my:443 "GET /search?keyword=&amp;property_type%5B%5D=1&amp;property_type%5B%5D=4&amp;property_type%5B%5D=5&amp;property_type%5B%5D=3&amp;state=kl_sel&amp;min_price=150000&amp;max_price=1000000&amp;min_size=&amp;max_size=&amp;sort=price-asc&amp;listing_status=upcoming&amp;page=1 HTTP/1.1" 200 None</msg>
<msg timestamp="20230523 14:56:46.924" level="DEBUG">Starting new HTTPS connection (1): www.lelongtips.com.my:443</msg>
<msg timestamp="20230523 14:56:47.687" level="DEBUG">https://www.lelongtips.com.my:443 "GET /search?keyword=&amp;property_type%5B%5D=1&amp;property_type%5B%5D=4&amp;property_type%5B%5D=5&amp;property_type%5B%5D=3&amp;state=kl_sel&amp;min_price=150000&amp;max_price=1000000&amp;min_size=&amp;max_size=&amp;sort=price-asc&amp;listing_status=upcoming&amp;page=1 HTTP/1.1" 200 None</msg>
<msg timestamp="20230523 14:56:47.794" level="INFO">now i am at page 1</msg>
<msg timestamp="20230523 14:56:47.795" level="INFO">G:\My Drive\Python\Scraper\Lelongtips\sherlock-dms\resources\restAPI\Main\Main.py:621: GuessedAtParserWarning: No parser was explicitly specified, so I'm using the best available HTML parser for this system ("lxml"). This usually isn't a problem, but if you run this code on another system, or in a different virtual environment, it may use a different parser and behave differently.

The code that caused this warning is on line 621 of the file G:\My Drive\Python\Scraper\Lelongtips\sherlock-dms\resources\restAPI\Main\Main.py. To get rid of this warning, pass the additional argument 'features="lxml"' to the BeautifulSoup constructor.

  parsed_html = BeautifulSoup(body_text)</msg>
<msg timestamp="20230523 14:56:47.795" level="FAIL">TypeError: int() argument must be a string, a bytes-like object or a real number, not 'list'</msg>
<msg timestamp="20230523 14:56:47.797" level="DEBUG">Traceback (most recent call last):
  File "G:\My Drive\Python\Scraper\Lelongtips\sherlock-dms\resources\restAPI\Main\Main.py", line 38, in user_runs_main_flow
    draft_content = self.get_pages(page_no_upper)
  File "G:\My Drive\Python\Scraper\Lelongtips\sherlock-dms\resources\restAPI\Main\Main.py", line 521, in get_pages
    content_list = self.get_contents(body_result)
  File "G:\My Drive\Python\Scraper\Lelongtips\sherlock-dms\resources\restAPI\Main\Main.py", line 632, in get_contents
    if int(re.findall("(\d+)", info)) &gt; 2:
TypeError: int() argument must be a string, a bytes-like object or a real number, not 'list'</msg>
<status status="FAIL" starttime="20230523 14:56:45.969" endtime="20230523 14:56:47.797"/>
</kw>
<doc>Executes main flow</doc>
<tag>flow</tag>
<status status="FAIL" starttime="20230523 14:56:45.968" endtime="20230523 14:56:47.797">TypeError: int() argument must be a string, a bytes-like object or a real number, not 'list'</status>
</test>
<doc>[https://pabot.org/?ref=log|Pabot] result from 1 executions.</doc>
<status status="FAIL" starttime="20230523 14:56:42.216" endtime="20230523 14:56:47.798"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tests</stat>
</total>
<tag>
<stat pass="0" fail="1" skip="0">flow</stat>
</tag>
<suite>
<stat pass="0" fail="1" skip="0" id="s1" name="Main">Main</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20230523 14:56:42.833" level="WARN">Imported library 'G:\My Drive\Python\Scraper\Lelongtips\sherlock-dms\resources\TransactionFormula.py' contains no keywords.</msg>
</errors>
</robot>
